Jenkinsfile

```diff
@@ -69,10 +69,16 @@ node('build-slave') {
             archiveArtifacts artifacts: 'metadata.json', onlyIfSuccessful: true
             currentBuild.description = artifact_version
         }
+        currentBuild.result = 'SUCCESS'
     }
 
     catch (err) {
-        currentBuild.result = "FAILURE"
+        currentBuild.result = 'FAILURE'
+        slackSend (
+            channel: "${env.BUILD_NOTIFY_SLACK_CHANNEL}",
```

Comment:
@Himanshu-optit Please perform the changes as below. We will have 5 channels which I discussed with Ganagram..
BUILD_NOTIFY_SLACK_CHANNEL
DEPLOY_NOTIFY_SLACK_CHANNEL
BACKUP_NOTIFY_SLACK_CHANNEL
OTHER_NOTIFY_SLACK_CHANNEL
GLOBAL_NOTIFY_SLACK_CHANNEL

so for all build jobs you need to perform a check like this 

```
if(env.BUILD_NOTIFY_SLACK_CHANNEL != null)
     code to send slack notification to this slack channel
    else if(env.GLOBAL_NOTIFY_SLACK_CHANNEL != null)
     code to send slack notification to this slack channel
    else
      println ANSI_YELLOW + ANSI_BOLD + "Could not find slack environment variable. Skipping slack notification.." + ANSI_NORMAL
```

For deploy jobs, please match the first if condition with DEPLOY_NOTIFY_SLACK_CHANNEL and so on.

Please make thi change to one of the Jenkinsfile first and let me know. Once the structure is finalized, you can update other Jenkinsfiles.

Reply:
@keshavprasadms , will make changes and update back